src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -28,6 +28,23 @@ import { LoggedInActionsQueryRenderer as LoggedInActions } from "./LoggedInActio
 import { NAV_BAR_HEIGHT } from "./constants"
 import { userHasLabFeature } from "v2/Utils/user"
 
+const NavBarContainer = styled(Flex)`
+  position: relative;
+  height: ${NAV_BAR_HEIGHT}px;
+  background-color: ${color("white100")};
+  border-bottom: 1px solid ${color("black10")};
+`
+
+const NavSection: React.FC<FlexProps> = ({ children, ...rest }) => {
+  return (
+    <Flex alignItems="stretch" height="100%" bg={rest.bg} {...rest}>
+      <Flex width="100%" height="100%" alignItems="center">
+        {children}
+      </Flex>
+    </Flex>
+  )
+}
+
```

Comment:
I just moved these up because it just makes more sense when reading it. I notice frequently that simple styled components are at the bottom of the file — which I find illegible/difficult to read? 🤷

Reply:
This is def a pref thing. The thinking is: when i open `NavBar.tsx`, i expect to immediately see `export const NavBar` definition. Anything intermediate is indirection.